docs/UPGRADE-4.0.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: page
+title: Upgrading from 3.x to 4.0
+---
+
+This document outlines the steps one should take when upgrading to the new major version.
+
+## Step 1: Review the new requirements
+
+Requirements have changed, this project now requires **PHP 7.4 or later** and **Moodle 3.8.3** or later.
+
+## Step 2: Review the change log
+
+Detailed information about what changed in Version 4 can be found in the [change log](CHANGELOG.md).
+
+## Step 3: Review the CIs configuration files
+
+### GitHub Actions
+
+Review the updated [gha.dist.yml](https://github.com/moodlehq/moodle-plugin-ci/blob/master/gha.dist.yml)
+and update the `.github/workflows/ci.yml` (or whatever it's named) file in your plugin. For detailed information about the contents of that `ci.yml` file, please see this [help document](GHAFileExplained.md).
+
+A summary of the actions required, really simple for version 4, is:
```

Comment:
I like the idea of celebrating the simplicity, but I also shy away from calling something "simple" if I'm giving someone else instructions (in case it's not simple for them).
```suggestion
A summary of the actions required for version 4 is:
```